ledger/participant-integration-api/src/main/scala/platform/apiserver/services/tracking/TrackerImpl.scala

```diff
@@ -64,6 +66,7 @@ private[services] final class TrackerImpl(queue: SourceQueueWithComplete[Tracker
     logger.debug("Shutting down tracking component.")
     queue.complete()
     Await.result(queue.watchCompletion(), 30.seconds)
+    Await.result(done, 30.seconds)
```

Comment:
I would love to use `Resource` or something to avoid `Await` on close.

Reply:
You would have to come up with an async version of the close method, i.e., `close(): Future[Unit]`.